lib/takeaway.rb

```diff
@@ -0,0 +1,39 @@
+class Takeaway
+  def initialize(menu)
+    @menu = menu
+    @basket = {}
+    @total = 0
+  end
+
+  attr_reader :basket, :total
+
+  def menu
+    @menu.dishes
+  end
+
+  def order(dish, number=1)
+    @basket[ordered(dish,number)] = price(dish,number)
+    @total += price(dish,number)
+  end
+
+  def basket_sum
+    sum = 0
+    @basket.each do |key, value|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)